tools/run_tests/run_interop_tests.py

```diff
@@ -635,10 +635,7 @@ def cloud_to_cloud_jobspec(language, test_case, server_name, server_host,
       '--server_host=%s' % server_host,
   ]
   if test_case in _HTTP2_BADSERVER_TEST_CASES:
-    # We are running the http2_badserver_interop test. Adjust command line accordingly.
-    offset = sorted(_HTTP2_BADSERVER_TEST_CASES).index(test_case)
-    client_options = common_options + ['--server_port=%s' %
-                                       (int(server_port)+offset)]
+    client_options = common_options + ['--server_port=%s' % server_port]
```

Comment:
Nit, but now that we don't handle offset here, you could include server port in common_options again. Might as well for readability. Also would suggest hoisting client_options out of this conditional, and then just doing a += in the block for interop_only_options